e2e/__projects__/basic/test.js

```diff
@@ -118,8 +118,19 @@ test('supports relative paths when extending templates from .pug files', () => {
 })
 
 test('supports class component .vue files', () => {
-  mount(ClassComponent, { msg: 'Hello' })
-  expect(document.querySelector('div').textContent).toBe('Hello')
+  expect.assertions(3)
+  mount(ClassComponent, { msg: 'Props Message' })
+  expect(document.querySelector('[data-computed]').textContent).toBe(
+    'Message: Props Message'
+  )
+  expect(document.querySelector('[data-props]').textContent).toBe(
+    'Props Message'
+  )
+  const event = new window.Event('click')
```

Comment:
is this not a nicer way to trigger an event?
```
wrapper.find('form').trigger('click');
```

Reply:
it would be, but library (intentionally) does not use Vue Test Utils :)

Reply:
Ah, didnt know it was a Vue Test Utils thing.